neuro_san/internals/network_providers/service_agent_network_storage.py

```diff
@@ -59,46 +59,46 @@ def remove_listener(self, listener: AgentStateListener):
         if listener in self.listeners:
             self.listeners.remove(listener)
 
-    def add_agent_tool_registry(self, agent_name: str, registry: AgentToolFactory):
+    def add_agent_network(self, agent_name: str, agent_network: AgentNetwork):
```

Comment:
Change nomenclature of access methods and args to be agent_network as opposed to "registry" and "tool_registry" and "tool_factory" 😱